README.md

```diff
@@ -97,7 +97,8 @@ Foundation](https://nixos.org/nixos/foundation.html). To ensure the
 continuity and expansion of the NixOS infrastructure, we are looking
 for donations to our organization.
 
-You can donate to the NixOS foundation by using Open Collective:
+You can donate to the NixOS foundation through [SEPA bank
+transfers](https://nixos.org/donate.html) or by using Open Collective:
```

Comment:
I think we prefer Open Collective even with the fee because it reduces the amount of accounting work we need to do. However, cc @rbvermaa who is treasurer of the foundation.

Reply:
we have it on the website, so i think we can link to it

if it shouldn't be used, it should be removed there to be consistent